doc/api/domain.markdown

```diff
@@ -308,56 +269,56 @@ than crashing the program.
 
 * {Array}
 
-An array of timers and event emitters that have been explicitly added
-to the domain.
+Returns an array of timers and `EventEmitter` objects that have been explicitly
+bound to the domain.
```

Comment:
The term "bound" or "bind" has a different meaning in the context of domains. Functions/callbacks are bound, event emitters/timers are added. `domain.members` contains only the list of added event emitters/timers, not the list of "bound" functions/callbacks.